_sass/settings/_variables.scss

```diff
@@ -4,10 +4,10 @@ $secondary-font: 'Gotham A', 'Gotham B' sans-serif;
 
 // Colors
 $dark-blue: #002131;
-$medium-blue: #0599e3;
-$light-blue: #46CAFF;
+$medium-blue: #007CBB;
```

Comment:
You're only changing values here, not variable names, but I'd suggest prefixing all of your colors to keep them consistent. A lot of the time we prefix color variables with `$c-`.

Also, names like "light blue" and "darker red" can be difficult to maintain. Because what if you want a red that's darker than `$red`, but lighter than `$darker-red`? To prevent this from happening, I like to use this color namer tool to name my variables for me: http://chir.ag/projects/name-that-color/#6195ED

Reply:
This is super helpful! Thanks